pm64/src/bgm/midi.rs

```diff
@@ -249,20 +259,27 @@ fn midi_track_to_bgm_track(
                                 if let Some(start) = started_notes.remove(&key) {
                                     let length = time - start.time;
 
-                                    track.commands.insert(
-                                        convert_time(start.time, time_divisor),
-                                        Command::Note {
-                                            pitch: key + 104,
-                                            velocity: start.vel,
-                                            length: convert_time(length as usize, time_divisor) as u16,
-                                        },
-                                    );
-
                                     if is_pitch_bent {
                                         is_pitch_bent = false;
-                                        track.commands.insert(
+                                        track.commands.insert_many(
                                             convert_time(time, time_divisor),
-                                            Command::SegTrackTune { coarse: 0, fine: 0 },
+                                            vec![
+                                                Command::SegTrackTune { bend: 0 },
+                                                Command::Note {
```

Comment:
Would be wise to do `let note = Command::Note { .. };` then refer to it here and the two times below as `note` (possibly with a clone if required)